src/azure-openapi-validator/rules/DescriptionMustNotBeNodeName.ts

```diff
@@ -0,0 +1,29 @@
+import { rules } from "../rule";
+
+rules.push({
+  id: "M2065",
+  name: "DescriptionMustNotBeNodeName",
+  severity: "error",
+  category: ["RPCViolation"],
+
+  appliesTo_JsonQuery: "$..*[?(@.description)]",
+  run: function* (doc, node, path) {
+    let msg: string = "Description must not match the name of the node it is supposed to describe";
```

Comment:
should we have the other properties as well? "openapi-type" and "changesimpact"? How should we keep in sync this with the properties that we have defined in the C# side?
Are we expecting to not write anymore rules on the C# side?